private-templates-service/client/src/components/Common/Tags/styled.tsx

```diff
@@ -15,7 +15,7 @@ export const TagCloseIcon = styled(Icon)`
   border-radius: 50%;
 
   &:hover {
-    font-family: ${FONTS.SEGOE_UI_SEMI_BOLD};
+    font-weight: 700;
```

Comment:
can these please be made `font-weight: bold` to be consistent with the styling of the rest of the app?